test/lib/usd/translators/testUsdExportDisplacement.py

```diff
@@ -0,0 +1,130 @@
+#!/pxrpythonsubst
+#
+# Copyright 2020 Pixar
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pxr import Usd
+from pxr import UsdShade
+
+from maya import cmds
+from maya import standalone
+
+import os
+import unittest
+
+import fixturesUtils
+
+
+class testUsdExportDisplacementShaders(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        inputPath = fixturesUtils.setUpClass(__file__)
+
+        mayaFile = os.path.join(inputPath, 'UsdExportDisplacementTest',
+            'SimpleDisplacement.ma')
+        cmds.file(mayaFile, force=True, open=True)
+
+        # Export to USD.
+        usdFilePath = os.path.abspath('SimpleDisplacement.usda')
+        cmds.mayaUSDExport(mergeTransformAndShape=True, file=usdFilePath,
+            shadingMode='useRegistry', convertMaterialsTo='UsdPreviewSurface',
+            materialsScopeName='Materials')
+
+        cls._stage = Usd.Stage.Open(usdFilePath)
+
+    @classmethod
+    def tearDownClass(cls):
+        standalone.uninitialize()
+
+    def testStageOpens(self):
+        '''
+        Tests that the USD stage was opened successfully.
+        '''
+        self.assertTrue(self._stage)
+
+    def testExportDisplacementShaders(self):
+        '''
+        Tests that exporting multiple Maya planes with varying displacement
+        setups results in the expected USD data:
+        '''
+
+        expected = [
+            # Mesh, Blinn, Image, Channel
```

Comment:
As you can see, the test scene exercises all connectable channels, using textures specifically designed to contain RGBA, RGB, RA, and R color data. This is more to exercise consumers than to exercise the conversion pipeline. Opening the scene in Maya, opening the exported USD file in usdView, and viewing the USD file on OS/X default viewer all give interesting and differing results.

Reply:
Yeah, I definitely understand that trying to get displacement to actually draw consistently across all possible apps/platforms is going to be an entirely different battle. :/